modules/ci/.github/workflows/github.tmpl

```diff
@@ -0,0 +1,52 @@
+name: Build and Deploy
```

Comment:
By default files will be templated and written to the same path that it resides in the package. This could be further customized by frontmatter params if needed.

So this file will be templated and written to projectname/.github/workflows/github.yml